line-bot-parser/build.gradle

```diff
@@ -0,0 +1,6 @@
+dependencies {
+    implementation project(':line-bot-api-client')
```

Comment:
Looks like parser & client relationship is a little difficult.

Can we make parser independent from client? (only depends for model.)